alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/endpoint/CustomEndpoint.java

```diff
@@ -35,6 +35,9 @@ public CustomEndpoint(String fieldKey, CustomEndpointManager customEndpointManag
         customEndpointManager.registerFunction(fieldKey, this::createResponse);
     }
 
+    /**
+     * @param fieldModel This FieldModel may be use for further computation in overriding classes
+     */
     protected Optional<ResponseEntity<String>> preprocessRequest(FieldModel fieldModel) {
```

Comment:
SonarCloud does not want a method to have unused parameters. In our case we do this so that overriding methods can use the parameter. The Javadoc allows SonarCloud to ignore this.